components/theme/blocks/AsyncSidebar/AsyncSidebar.vue

```diff
@@ -94,7 +94,7 @@ export default {
 .right-sidebar {
   top: 0;
   right: 0;
-  z-index: 4;
+  z-index: 1001;
```

Comment:
Couldn't we have a lower value?

Reply:
Back to top have `z-index: 1000`

![Zrzut ekranu 2020-11-12 o 11 40 40](https://user-images.githubusercontent.com/40273258/98929771-f0b89b00-24db-11eb-934c-50784091f12d.png)